web/Migrations/20220726103731_AddInviteLinkToLandlord.cs

```diff
@@ -0,0 +1,188 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BricksAndHearts.Migrations
+{
+    public partial class AddInviteLinkToLandlord : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
+                name: "TownOrCity",
+                table: "Property",
+                type: "nvarchar(max)",
+                nullable: true,
+                oldClrType: typeof(string),
+                oldType: "nvarchar(max)");
```

Comment:
These extra alters shouldn't be here. Possibly there was a missing migration on main at some point? Will hopefully be fixed by rebasing and recreating this migration

Reply:
yes I've rebased onto main and it is now all clean.